repository_service_tuf/cli/artifact/repository.py

```diff
@@ -0,0 +1,278 @@
+# SPDX-FileCopyrightText: 2023 VMware Inc
+#
+# SPDX-License-Identifier: MIT
+
+
+import base64
+
+from click import Context
+from dynaconf.loaders.yaml_loader import write  # type: ignore
+
+from repository_service_tuf.cli import click, console
+from repository_service_tuf.cli.artifact import artifact
+
+
+def write_config(settings_path, settings_data, merge=True):
+    write(settings_path, settings_data, merge)  # pragma: no cover
```

Comment:
What error did you stop with `pragma`?

Reply:
"no cover", i.e. a test coverage check, as it's an external method we don't want to test, but reduces codecov %.